packages/react/src/internal/utils/getInteractiveNodes.ts

```diff
@@ -0,0 +1,23 @@
+const interactiveElements = [
+  'a[href]',
+  'button:not([disabled])',
+  'summary',
+  'select',
+  'input:not([type=hidden])',
+  'textarea',
+  '[tabindex="0"]',
+]
+
+export function getInteractiveNodes(node: HTMLElement | null, ignoreSelectors?: string) {
```

Comment:
One style suggestion, since we're returning a `boolean` this could be `hasInteractiveNodes` as opposed to `getInteractiveNodes` which might seem like we are returning the list of interactive nodes within the given parent node.

```suggestion
export function hasInteractiveContent(node: HTMLElement | null, ignoreSelectors?: string): boolean {
```

Reply:
That's a good callout. I changed the names of the function + file!